datanode/wrap_operator.go

```diff
@@ -450,6 +454,19 @@ func (s *DataNode) handleUpdateVerPacket(p *repl.Packet) {
 
 }
 
+func (s *DataNode) checkVolumeForbidden(volNames []string) {
+	s.space.RangePartitions(func(partition *DataPartition) bool {
```

Comment:
check if volNames if empty first, if empty, return directly, no need to range partitions.

Reply:
No, we cannot do it.

We need it to quit forbidden status.